README.md

```diff
@@ -83,6 +83,6 @@ client.on('messageCreate', message => {
        embed.setFooter(message.author.username, message.author.avatarURL);
        embed.setTimestamp();
     
-       message.channel.createMessage({files: [Attachment], embeds: [embed] });
+       message.channel.createMessage({file: Attachment, embeds: [embed] });
```

Comment:
i evaluated a little and the correct form it would be like this
~~~javascript
message.channel.createMessage({embeds: [embed]}, {file: Attachment, name: 'file.png'});
~~~

please, edit this part and return me

Reply:
I did a better dipping, you are right indeed, mine does work in some exceptions, but this is the eris standard, you just don't need to put keys on it.